### PR TITLE
Fix pubsub errors

### PR DESCRIPTION
**Pull request by KutnerUri**
## Proposed Changes

- update pubsub 
- the update fixes the `[Pubsub.ui] failed connecting to child iframe` error happening when switching between component pages, see [here](https://github.com/Aaronius/penpal/releases/tag/v6.0.0).
- turn off pubsub for gallery pages
- remove the unused hotReload flag that also trigger errors

### CODE REVIEW

File: scopes/compositions/compositions/ui/composition-preview.tsx [hunk 1]
```diff
@@ -1,60 +1,27 @@
...
+      {...rest}
       style={{ width: '100%', height: '100%' }}
       previewName="compositions"
       queryParams={compositionParams}
-      hotReload={hotReload}
     />
   );
 }
-
-ComponentComposition.defaultProps = {
```
Comment (odedre) [inline scopes/compositions/compositions/ui/composition-preview.tsx:58]: dont we need this anymore?

File: workspace.jsonc [hunk 2]
```diff
@@ -282,7 +282,7 @@
         "parallel-webpack": "2.6.0",
         "parse-package-name": "0.1.0",
         "path-to-regexp": "6.2.0",
-        "penpal": "5.3.0",
+        "penpal": "6.2.1",
```
Comment (odedre) [inline workspace.jsonc:285]: did we check that this update doesnt break stuff?

**Review by odedre — APPROVED** [review]

File: workspace.jsonc (hunk 2 above)
Comment (KutnerUri) [reply workspace.jsonc:285]: yes I did

File: scopes/compositions/compositions/ui/composition-preview.tsx (hunk 1 above)
Comment (KutnerUri) [reply scopes/compositions/compositions/ui/composition-preview.tsx:58]: yes, the hotReload flag was not working for a while

**Pull request merged**